Enterprise-CMCS/macpro-mdct-mfp
# [CMDCT-3366] - Unskip and split up accessibility tests

### Description

Accessibility tests should now be running for MFP! 

### Related ticket(s)


CMDCT-3366

---
### How to test

Note the password is the same for logging in as a state user or admin user in MFP. If you don't have the password or think its been updated, check the proj-cms-mdct-devs channel for the pinned message at the top called MDCT App Access
- Navigate to tests/cypress
- Run `CYPRESS_ADMIN_USER_PASSWORD='insertpasswordhere!' CYPRESS_STATE_USER_PASSWORD='Insertpasswordhere!' yarn test`
- Click e2e configuration
- Run the new tests! 

### Important updates


Added a slick testPageAccessibility command to reduce the amount of copied code. Can use for any page!

---
### Author checklist


---
